### PR TITLE
added column-2 in index and its css in css/section-2.css

### PR DESCRIPTION
**Pull request by optimisticlucifer**
Sorry I  directly started working please review my work.
I have attached the screenshot .
Screenshot 2021-06-20 at 2 46 49 PM

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -167,7 +168,21 @@ <h2 class="heading-36"> About us </h2>
           </div>
           <div id="about-part-2">
             <!-- Add column 2 here / TODO -->
-            Column 2
+            <p class="who">WHO</p>
```
Comment (Signior-X) [inline index.html:171]: Nice work, easy to understand classes.

File: index.html [hunk 2]
```diff
@@ -43,6 +43,7 @@
   <link rel="stylesheet" href="css/grid.css">
   <link rel="stylesheet" href="css/header.css">
   <link rel="stylesheet" href="css/section-1.css">
+  <link rel="stylesheet" href="css/section-2.css">
```
Comment (Signior-X) [inline index.html:46]: Do not add another css file

File: css/section-2.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
+.who {
```
Comment (Signior-X) [inline css/section-2.css:1]: Add all these changes in `about.css`

File: css/section-2.css [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  color: #b35efe;
+  font-family: "Open Sans", sans-serif;
+  font-size: 1rem;
+  font-weight: bolder;
+}
+#about-part-2 h1 {
+  color: #292c34;
+  font-family: "Open Sans", sans-serif;
```
Comment (Signior-X) [inline css/section-2.css:9]: No need to write font family anwhere, it is already defined in `main.css`

File: css/section-2.css [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  font-size: 4rem;
+  margin-block-start: 1rem;
+  margin-block-end: 1rem;
+}
+
+#about-part-2 h1::after {
+  content: "";
+  width: 100%;
+  background-color: #b35efe;
```
Comment (Signior-X) [inline css/section-2.css:18]: Change it to 
`background-color: var(--color-denim);`
`background-color: var(--color-denim--dark);`

Like some color matching with the color scheme, as you can see in `main.css`

**Review by Signior-X — CHANGES_REQUESTED** [review]
Nice work man, and please from the next time, dont forget to comment on the issue. I have reviewed your PR. Make changes, then it will be good to merge.

File: index.html (hunk 2 above)
Comment (optimisticlucifer) [reply index.html:46]: okay sir i will take care from next time.

File: css/section-2.css (hunk 3 above)
Comment (optimisticlucifer) [reply css/section-2.css:2]: okay i will make change then push it again

File: css/section-2.css (hunk 4 above)
Comment (optimisticlucifer) [reply css/section-2.css:10]: okay

File: css/section-2.css (hunk 5 above)
Comment (optimisticlucifer) [reply css/section-2.css:19]: 😀

Comment (optimisticlucifer) [reply css/section-2.css:19]: sir After changing colour its look like this so i dont change.😇
Screenshot 2021-06-20 at 4 42 32 PM

**Comment by optimisticlucifer** [comment]
sir i tried to make column 1 and 2 both and push in same developer branch.
i have attached my ss please review
Screenshot 2021-06-20 at 4 45 01 PM

**Comment by Signior-X** [comment]
Great work @optimisticlucifer thanks for this. Will merge this soon. 
Just some suggestions from my side, always comment on the issue you want to work. And different issues should be solved in different PRs. Rest nice work done :tada:

**Comment by Signior-X** [comment]
Specially for the left column, nice work done. I hope so you learned a lot. Nice way designed.

**Comment by Signior-X** [comment]
I have squashed and merged your PR, it will show in the code soon.

**Pull request closed without merging**